pkg/api/k8s_versions.go

```diff
@@ -10,412 +10,137 @@ import (
 	"github.com/Azure/aks-engine/pkg/api/common"
 )
 
-// k8sComponentVersions defines "best known-working" component versions for each minor version of k8s
```

Comment:
This func comment is not true (perhaps it was at some point). In fact the only reason to keep this additional layer of indirection is to aid some UT implementations.